examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -44,6 +48,37 @@ class Task extends TypedEventEmitter<ITaskEvents> implements ITask {
             this.emit("priorityChanged");
         });
     }
+    private readonly resetDiffProperties = (): void => {
+        this.diffPriority = DEFAULT_DIFF_PRIORITY;
```

Comment:
if these are sourceName and sourcePriority, then there's no need to reset them (they will remain truthfully reporting the source values).